src/main/java/pl/edu/pw/elka/paprykaisalami/geeruh/issues/domain/models/Timestamp.java

```diff
@@ -0,0 +1,12 @@
+package pl.edu.pw.elka.paprykaisalami.geeruh.issues.domain.models;
+
+import lombok.NonNull;
+import lombok.Value;
+
+import java.util.Date;
+
+@Value(staticConstructor = "of")
+public class Timestamp {
+    @NonNull
+    Date value;
+}
```

Comment:
I think it doesn't serve any purpose to wrap Date into Timestamp. There are 2 reasons for making value classes that i can see:
1. so that you won't mix up two things which have the same underlying type.
2. so that you can put some rules into value class, e.g. e-mail should have '@' somewhere inside, or quantity of something cannot be negative.

Timestamp here serves neither.

Just for clarification - probably I also could have omitted `Title` and `Description` value classes, because right now they aren't really useful. They only serve the first purpose here, but tbh I don't think it's very important in this specific situation.